templates/widget/README.md

```diff
@@ -0,0 +1,7 @@
+# Example widget
+
+The example extension demonstrates how to contribute a custom widget (i.e. a view or editor) to Eclipse Theia.
+
+## How to use the widget example
+
+In the running application, open the widget using the menu "View" => "<%= params.extensionPrefix %> Widget"
```

Comment:
Did you confirm that we are able to substitute the template with the widget name in markdown?
Else we might go for a simpler initial documentation by using an example for instance.

Reply:
Yes, I explicitly tested that!